api/envoy/config/filter/network/thrift_proxy/v2alpha1/route.proto

```diff
@@ -95,6 +96,11 @@ message RouteAction {
   // N.B. Thrift service or method name matching can be achieved by specifying a RequestHeaders
   // action with the header name ":method-name".
   repeated api.v2.route.RateLimit rate_limits = 4;
+
+  // Strip the service prefix from the method name, if there's a prefix. For
+  // example, the method call Service:method would end up being just method.
+  // Defaults to false.
```

Comment:
No need to mention the default if it's a native proto type, as that is implied. Only need to do this for wrapped types.

Reply:
fixed.